alot/db/message.py

```diff
@@ -107,13 +107,11 @@ def get_thread_id(self):
         return self._thread_id
 
     def get_message_parts(self):
-        """returns a list of all body parts of this message"""
+        """yield all body parts of this message"""
         # TODO really needed? email  iterators can do this
```

Comment:
I didn't see how any of [this](https://docs.python.org/2/library/email.iterators.html) fits the bill. So maybe we should remove the comment? Or am I missing something?